### ISSUE TITLE
23376704: UIBarButtonItem's accessibilityLabel should be settable through interface builder

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#6634]
#### Description

Summary:

UIBarButtonItem's accessibilityLabel cannot be set through interface builder, meaning these have to be set in code.

Steps to Reproduce:
1. In interface builder, create a bar button item
2. Go to the Identity Inspector tab

Expected Results:

There should be a section for adding accessibility labels, identifiers, etc.

Actual Results:

There is no way to change these accessibility related properties through interface builder.

Notes:

Currently we are working around this using this extension on UIBarButtonItem:

public extension UIBarButtonItem {
    /// UIBarButtonItem accessibilityLabel settable from IB
    @IBInspectable public var voiceoverLabel: String! {
        get { return self.accessibilityLabel }
        set { self.accessibilityLabel = newValue }
    }
}
## 

Product Version: Xcode 7.1
Created: 2015-11-03T18:08:02.114670
Originated: 2015-11-03T10:07:00
Open Radar Link: http://www.openradar.me/23376704
